input/resources/au-relatedperson.xml

```diff
@@ -49,8 +49,8 @@
       <constraint>
           <key value="inv-relper-4" />
           <severity value="warning" />
-          <human value="Recorded sex or gender jurisdiction shall be a member of the AU Recorded Sex or Gender (RSG) Source Document Jurisdiction value set if any codes within that value set can apply" />
-        <expression value="extension('http://hl7.org/fhir/StructureDefinition/individual-recordedSexOrGender').extension('jurisdiction').empty() or extension('http://hl7.org/fhir/StructureDefinition/individual-recordedSexOrGender').all(extension('jurisdiction').value.memberOf('http://terminology.hl7.org.au/ValueSet/rsg-source-document-jurisdiction'))"/>
+          <human value="Recorded sex or gender jurisdiction shall be a member of the Jurisdiction ValueSet - AU Extended if any codes within that value set can apply" />
```

Comment:
IG publisher isn't recognising the url with that case. Replace with "ValueSet-au-jurisdiction-extended.html"

Reply:
Updated valueset page name to Title case